src/app/app.component.css

```diff
@@ -64,4 +66,6 @@ body {
     --button-text2: #4285F4;
     --table-header: #d4d4d6;
     --scrollbar-color: rgba(255, 255, 255, 0.51);
+    --togglebutton-color: #29382c;
+    --text-color: #ccc4c4;

```

Comment:
Use Global variables for test color, and button toggle color

Reply:
Resolved changes